### PR TITLE
Enable hypervisor after RL9 compute migration

### PR DESCRIPTION
**Pull request by MoteHue**

### CODE REVIEW

**Review by Alex-Welsh — DISMISSED** [review]

File: doc/source/operations/rocky-linux-9.rst [hunk 1]
```diff
@@ -456,6 +456,13 @@ address them.
 Once happy that the system has been restored to full health, move onto the next
 host or batch or hosts.
 
+9. Enable the hypervisor in Nova again:
+
+   .. code-block:: console
+
+      kayobe playbook run $KAYOBE_CONFIG_PATH/ansible/nova-compute-enable.yml --limit <host>
```
Comment (Alex-Welsh) [inline doc/source/operations/rocky-linux-9.rst]:
```suggestion
      kayobe playbook run $KAYOBE_CONFIG_PATH/ansible/nova-compute-enable.yml --limit <hostname>
```
To match the rest of the doc

File: doc/source/operations/rocky-linux-9.rst [hunk 2]
```diff
@@ -456,6 +456,13 @@ address them.
 Once happy that the system has been restored to full health, move onto the next
 host or batch or hosts.
```
Comment (markgoddard) [inline doc/source/operations/rocky-linux-9.rst]: The indentation suggests this part should be after the numbered list.

File: doc/source/operations/rocky-linux-9.rst [hunk 3]
```diff
@@ -456,6 +456,13 @@ address them.
...
 
+9. Enable the hypervisor in Nova again:
+
+   .. code-block:: console
+
+      kayobe playbook run $KAYOBE_CONFIG_PATH/ansible/nova-compute-enable.yml --limit <hostname>
```
Comment (markgoddard) [inline doc/source/operations/rocky-linux-9.rst]: I wonder why you are the first to see this. Perhaps the service deploy sometimes brings it up enabled, depending on timing?

**Review by markgoddard — CHANGES_REQUESTED** [review]

File: doc/source/operations/rocky-linux-9.rst (hunk 3 above)
Comment (MoteHue) [reply doc/source/operations/rocky-linux-9.rst]: I've only migrated one hypervisor (the rest are waiting for the az team to do), so I'm not sure about the timings. I can edit this to suggest checking if it is disabled first? Although I think running the enable on a hypervisor that's already enabled won't do anything.

File: doc/source/operations/rocky-linux-9.rst (hunk 2 above)
Comment (MoteHue) [reply doc/source/operations/rocky-linux-9.rst]: I'd put this step after as we wouldn't want to enable it again until we are "happy that the system has been restored to full health". I'll edit to make this clearer

Comment (markgoddard) [reply doc/source/operations/rocky-linux-9.rst]: How would we know it's been restored to full health if it's disabled?

Comment (MoteHue) [reply doc/source/operations/rocky-linux-9.rst]: I interpreted this more as the migrated hypervisor(s) are configured and running all the services and no errors/alerts are firing.

File: doc/source/operations/rocky-linux-9.rst [hunk 4]
```diff
@@ -448,13 +448,21 @@ Full procedure for one batch of hosts
 
        kayobe playbook run $KAYOBE_CONFIG_PATH/ansible/wazuh-agent.yml -l <hostname>
 
-If any VMs were powered off, they may now be powered back on.
+9. Restore the system to full health.
 
-Wait for Prometheus alerts and errors in OpenSearch Dashboard to resolve, or
-address them.
+   1. If any VMs were powered off, they may now be powered back on.
```
Comment (markgoddard) [inline doc/source/operations/rocky-linux-9.rst:453]: Can you power on VMs on a disabled compute service?

File: doc/source/operations/rocky-linux-9.rst (hunk 3 above)
Comment (markgoddard) [reply doc/source/operations/rocky-linux-9.rst]: Right - it should be idempotent

File: doc/source/operations/rocky-linux-9.rst (hunk 4 above)
Comment (MoteHue) [reply doc/source/operations/rocky-linux-9.rst:453]: Yes

**Review by markgoddard — APPROVED** [review]

**Pull request merged**